src/ck-pics/picsautoperfAPI.h

```diff
@@ -7,8 +7,26 @@
     extern "C" {
 #endif
 
+struct PicsConfig {
+  bool fromGlobal;
+  int numPhases;
+  int collectionMode;
+  int evaluationMode;
+  int warmupSteps;
+  int pauseSteps;
+  char **phaseNames;
+  PicsConfig() {
+    fromGlobal = true;
+    numPhases = PICS_INVALID;
+    collectionMode = PICS_INVALID;
+    evaluationMode = PICS_INVALID;
+    warmupSteps = PICS_INVALID;
+    pauseSteps = PICS_INVALID;
+    phaseNames = nullptr;
+  }
```

Comment:
We require C++11 support, so you can safely use default member initializers instead of a constructor method.

Reply:
Ok, done. Thanks!